travis/test_server.py

```diff
@@ -279,6 +279,13 @@ def hidden_line(line, main_modules, addons_path_list=None,
             # hidden all schema debug except
             #   dropped column, changed type and changed size
             return True
+    if 'superfluous' in line and 'dependency' in line:
+        superfluous_regex = re.compile(r'\.(?P<module>[\d\w$_]+): ')
+        superfluous_regex_search = superfluous_regex.search(line)
+        if superfluous_regex_search:
+            module = superfluous_regex_search.group('module')
+            if module not in main_modules:
+                return True
```

Comment:
From odoo I will create you a report to get all superfluous dependencies:

![from odoo](https://cloud.githubusercontent.com/assets/6644187/16136659/1d30b6f2-33f2-11e6-8e7e-b255016c7673.png)


From log we should show just the modules of `app` project and `addons-vauxoo` in order to continue with the current process of MQT to get log analysis of modules by project.